roles/confluent.common/tasks/main.yml

```diff
@@ -1,4 +1,13 @@
 ---
+- name: Gather OS Facts
+  setup:
+    filter: "{{item}}"
+    gather_subset:
+      - '!all'
```

Comment:
Might be worth while to add a comment stating this ignores all facts, and only gathers what is specified in the filter.